src/Testcontainers.Ollama/OllamaContainer.cs

```diff
@@ -0,0 +1,52 @@
+namespace Testcontainers.Ollama
+{
+    /// <inheritdoc cref="DockerContainer" />
+    [PublicAPI]
+    public sealed class OllamaContainer : DockerContainer
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OllamaContainer" /> class.
+        /// </summary>
+        /// <param name="configuration">The container configuration.</param>
+        /// <param name="logger">The logger.</param>
+        public OllamaContainer(OllamaConfiguration configuration, ILogger logger)
+            : base(configuration, logger)
+        {
+            ModelName = configuration.ModelName;
+            ImageName = OllamaConfiguration.ImageName;
+        }
+    
+        /// <summary>
+        /// Starts the Ollama container.
+        /// </summary>
+        public async Task StartOllamaAsync()
+        {
+            if (State!= TestcontainersStates.Created && State != TestcontainersStates.Running)                {
+                throw new InvalidOperationException("Cannot start a container that has not been created.");
+            }
```

Comment:
We use `ThrowIfResourceNotFound();` to make sure the resources exists and is "running".

Reply:
Done